### ISSUE TITLE
md5sum missing and abundances 

### ISSUE THREAD

**Issue by EduardoCristo** [ideasrule/platon#44]
When running the example emcee file:

 File "platon_p.py", line 56, in <module>
    result = retriever.run_emcee(bins, depths, errors,None, None, None,fit_info)
  File "/home/user/anaconda3/lib/python3.7/site-packages/platon/combined_retriever.py", line 248, in run_emcee
    include_condensation=include_condensation, method=rad_method)
  File "/home/user/anaconda3/lib/python3.7/site-packages/platon/transit_depth_calculator.py", line 44, in __init__
    ref_pressure, method)               
  File "/home/user/anaconda3/lib/python3.7/site-packages/platon/_atmosphere_solver.py", line 30, in __init__
    get_data_if_needed()
  File "/home/user/anaconda3/lib/python3.7/site-packages/platon/_get_data.py", line 16, in get_data_if_needed
    with open(resource_filename(__name__, "md5sum")) as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/user/anaconda3/lib/python3.7/site-packages/platon/md5sum'

Thank you for the nice code!

**Comment by ideasrule** [comment]
This occurs when the data extraction is cancelled halfway through, leaving a data directory but no md5sum file.  Can you delete the data directory:

rm -r /home/user/anaconda3/lib/python3.7/site-packages/platon/data

and try again?

Thanks for using PLATON!

**Comment by EduardoCristo** [comment]
Thank you very much, it worked.
I would like to congratulate for this nice piece of work. 
One more question: Is it possible to have as output a list of detected species and there abundaces? 

Cheers,
Eduardo.

**Comment by ideasrule** [comment]
Hi Eduardo,

Unfortunately there's no built-in way to do this, but here's how I did it in my paper: https://arxiv.org/pdf/2004.09513.pdf

I first did a retrieval with all molecules included.  I looked at the best fit solution, and removed absorption files from PLATON_DIR/platon/data/Absorption that don't make a difference at all (which I define as decreasing chi^2 by more than 1).  Usually this gets you down to a handful of molecules.  To see whether these remaining molecules are robustly detected, I run a retrieval with this handful of of molecules, and then run another retrieval with one of them removed (so delete another opacity file from PLATON_DIR/platon/data/Absorption).  I compare final_logZ, the ln of the Bayesian evidence, and consider the molecule detected if it drops by more than 2 after the molecule is removed.


Hope this helps,

Michael
